umodbus/client/tcp/asynch.py

```diff
@@ -0,0 +1,29 @@
+"""
```

Comment:
I think the filename is `asynch.py` as opposed to `async.py` because `async` is a protected keyword, right? Would be nice if we could call it `async` but I guess if Python doesn't let us, this is the next best thing.

Reply:
Yes, exactly.